packages/transmute-cli/src/commands/init.js

```diff
@@ -1,12 +1,17 @@
 const run = require('./runner');
 
-export function k8s( dryrun, clusterName ) {
-  let prov_cmd = "$HOME/.transmute/git/transmute/scripts/initializer " + clusterName;
-  console.log( prov_cmd );
-  if (dryrun == 'true' ) {
-    console.log( '<--dry run-->' );
-  }
-  else {
-    run.ner( prov_cmd );
+const path = require('path');
+
+module.exports.k8s = (dryrun, clusterName) => {
+  let prov_cmd = path.join(
+    __dirname,
+    '../../scripts/initializer ' + clusterName
+  );
+
+  // console.log( prov_cmd );
+  if (dryrun == 'true') {
+    console.log('<--dry run-->');
```

Comment:
Do we want to start using `console.info` or `console.debug` for things like this to make them stand out?

Reply:
we want to use winston and a proper logger. until then, console logging is fine.